UnityAds/src/main/java/com/applovin/mediation/adapters/UnityAdsMediationAdapter.java

```diff
@@ -146,14 +151,6 @@ public void loadInterstitialAd(final MaxAdapterResponseParameters parameters, fi
         String placementId = parameters.getThirdPartyAdPlacementId();
         log( "Loading " + ( AppLovinSdkUtils.isValidString( parameters.getBidResponse() ) ? "bidding " : "" ) + "interstitial ad for placement \"" + placementId + "\"..." );
 
-        if ( !UnityAds.isInitialized() )
```

Comment:
This explains a lot

Reply:
Do we want to remove these for all types of Ads?

Reply:
yes